source/include/TDCodecVia.h

```diff
@@ -223,6 +224,7 @@ void Format(SubString *format, Int32 count, StaticTypeAndData arguments[], Strin
 #define tsParamBlockTypeToken   "p"  // Used for defining param blocks used by native functions.
 #define tsPointerTypeToken      "ptr"
 #define tsRefNumTypeToken       "refnum"
+#define tsEnumTypeToken         "Enum"
```

Comment:
By looking at the other elements in the list does it make sense to have "enum", instead of "Enum"?